Library/Homebrew/formula.rb

```diff
@@ -1288,9 +1290,18 @@ def system(cmd, *args)
           end
           wr.close
 
-          while buf = rd.gets
-            log.puts buf
-            puts buf
+          if verbose
+            while buf = rd.gets
+              log.puts buf
+              puts buf
+            end
+          else # travis
+            while buf = rd.gets
+              log.puts buf
+              print "."
```

Comment:
Yeh, I like the idea of handling build freezing too. I think that printing every line is a good idea but will probably print too many dots. So, instead of printing a dot every 1 minute it prints a dot on every line unless there was already a dot printed in the last minute. How does that sound?